MLTAscend/MLTAscend.MVC/Views/Shared/_Layout.cshtml

```diff
@@ -1,77 +1,73 @@
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+          <span aria-hidden="true"></span>
+        </a>
+      </div>
+
+      <div id="navbarBasicExample" class="navbar-menu">
+        <div class="navbar-start">
+          <a class="navbar-item">Home</a>
+          <a class="navbar-item">Documentation</a>
+
+          <div class="navbar-item has-dropdown is-hoverable">
+            <a class="navbar-link">More</a>
 
-    <environment include="Development">
-        <script src="~/lib/jquery/dist/jquery.js"></script>
-        <script src="~/lib/bootstrap/dist/js/bootstrap.bundle.js"></script>
-    </environment>
-    <environment exclude="Development">
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/jquery/3.3.1/jquery.min.js"
-                asp-fallback-src="~/lib/jquery/dist/jquery.min.js"
-                asp-fallback-test="window.jQuery"
-                crossorigin="anonymous"
-                integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8=">
-        </script>
-        <script src="https://cdnjs.cloudflare.com/ajax/libs/twitter-bootstrap/4.1.3/js/bootstrap.bundle.min.js"
-                asp-fallback-src="~/lib/bootstrap/dist/js/bootstrap.bundle.min.js"
-                asp-fallback-test="window.jQuery && window.jQuery.fn && window.jQuery.fn.modal"
-                crossorigin="anonymous"
-                integrity="sha256-E/V4cWE4qvAeO5MOhjtGtqDzPndRO1LBk8lJ/PR7CA4=">
-        </script>
-    </environment>
-    <script src="~/js/site.js" asp-append-version="true"></script>
+            <div class="navbar-dropdown">
+              <a class="navbar-item" asp-area="" asp-controller="Home" asp-action="Index">Home</a>
+              <a class="navbar-item" asp-area="" asp-controller="Home" asp-action="Privacy">Privacy</a>
+              <a class="navbar-item">Contact</a>
+              <hr class="navbar-divider">
+              <a class="navbar-item">Report an issue</a>
+            </div>
+          </div>
+        </div>
 
-    @RenderSection("Scripts", required: false)
+        <div class="navbar-end">
+          <div class="navbar-item">
+            <div class="buttons">
+              <a class="button is-primary" asp-controller="User" asp-action="SignUp"><strong>Sign up</strong></a>
+              <a class="button is-light" asp-controller="User" asp-action="LogIn">Log in</a>
+            </div>
+          </div>
+        </div>
+      </div>
+    </nav>
+  </header>
+  <section class="section">
+    <div class="container">
+      <h1 class="title">Hello World</h1>
+      <p class="subtitle">My first website with <strong>Bulma</strong>!</p>
+      @RenderBody()
+    </div>
+  </section>
+  <footer class="footer">
+    <div class="content has-text-centered">
+      <p>
+        <strong>MLT Ascend</strong> was created by Jacob Convington, Mason Degeneffe and Nathan Johnson.
```

Comment:
"Covington" not "Convington" 👎

Reply:
![image](https://user-images.githubusercontent.com/45204676/52075645-d148bc00-255a-11e9-9bf2-3e635784b9b5.png)